docs: add missing ng update command to v19 update guide

aboudard opened the pull request:
This change adds the command to run when updating angular project from v18 to v19

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.dev application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
The ng update command is missing from the official update guide, from v18 to v19

Issue Number: N/A


## What is the new behavior?
The line is added to the recommendations.ts file

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

JeanMeche commented on adev/src/app/features/update/recommendations.ts:
```diff
@@ -2452,6 +2452,14 @@ export const RECOMMENDATIONS: Step[] = [
     possibleIn: 1900,
     step: '19.0.0.1',
   },
+  {
```
We probably should move that item a bit higher up on the list so it appear first. It should go before `Angular directives, compone ...`.

aboudard replied:
> We probably should move that item a bit higher up on the list so it appear first. It should go before `Angular directives, compone ...`.

Yeah, I placed it just before the TypeScript upgrade, but I guess it could better.

aboudard replied:
Ah ... I pushed with another git username, and I don't intend to make the CLA for the other one.
I might just delete the PR and make another one, what do you think ?

aboudard committed:
docs: add missing ng update command to v19 update guide

This change adds the command to run when updating angular project from v18 to v19

JeanMeche reviewed (approved):
Thanks for this Alain ;)

AndrewKushnir commented:
This PR was merged into the repository by commit 466d5579833f505928342ef45a90ca5237fba4a2.

The changes were merged into the following branches: main, 19.0.x

aboudard commented:
> Thanks for this Alain ;)

My pleasure Matthieu :)
